definitions/v1/abnormal.yml

```diff
@@ -0,0 +1,129 @@
+---
+id: abnormal
+name: Abnormal
+description: "General French Private Tracker"
+language: fr-fr
+type: private
+encoding: UTF-8
+links:
+  - https://abn.lol/ # site does not support https ERR_CONNECTION_CLOSED
```

Comment:
comment of not supporting https does not match the fact that it is an https link?

Reply:
Oh, it was a bad comment, forgot to remove it